sql/patch.sql

```diff
@@ -55,3 +55,7 @@ UPDATE list_options SET option_id='DOB' WHERE list_id='recent_patient_columns' A
 #IfMissingColumn form_encounter last_update
 ALTER TABLE `form_encounter` ADD `last_update` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP;
 #EndIf
+
+#IfMissingColumn form_encounter ordering_provider_id
+ALTER TABLE `form_encounter` ADD `ordering_provider_id` INT(11) DEFAULT '0' COMMENT 'ordering provider, if any, for this visit';
+#EndIf
```

Comment:
also need this in `database.sql`?

Reply:
We don't modify production database so that installing DB below the patch level could break.
DB changes should always be at patch level version via patch.sql.

Reply:
But this is rel-702 as well so need it.

Reply:
hi , reason we also do database.sql is to support new installs. Need for dockers (since we update docker with most recent patch) and when we need to rebuild the packages from a patch.